tst/EnergyPlus/unit/HVACVariableRefrigerantFlow.unit.cc

```diff
@@ -1881,18 +2487,18 @@ TEST_F( HVACFixture, VRFTest_SysCurve_WaterCooled ) {
 		DataSizing::CurZoneEqNum = 1;
 		DataEnvironment::OutBaroPress = 101325; // sea level
 		DataZoneEquipment::ZoneEquipInputsFilled = true; // denotes zone equipment has been read in
-		StdRhoAir = PsyRhoAirFnPbTdbW( DataEnvironment::OutBaroPress, 20.0, 0.0 );
-		ZoneEqSizing.allocate( 1 );
-		ZoneSizingRunDone = true;
-		ZoneEqSizing( CurZoneEqNum ).DesignSizeFromParent = false;
-		ZoneEqSizing( CurZoneEqNum ).SizingMethod.allocate( 25 );
-		ZoneEqSizing( CurZoneEqNum ).SizingMethod( DataHVACGlobals::SystemAirflowSizing ) = DataSizing::SupplyAirFlowRate;
+		DataEnvironment::StdRhoAir = PsyRhoAirFnPbTdbW( DataEnvironment::OutBaroPress, 20.0, 0.0 );
+		DataSizing::ZoneEqSizing.allocate( 1 );
+		DataSizing::ZoneSizingRunDone = true;
+		DataSizing::ZoneEqSizing( CurZoneEqNum ).DesignSizeFromParent = false;
+		DataSizing::ZoneEqSizing( CurZoneEqNum ).SizingMethod.allocate( 25 );
+		DataSizing::ZoneEqSizing( CurZoneEqNum ).SizingMethod( DataHVACGlobals::SystemAirflowSizing ) = DataSizing::SupplyAirFlowRate;
```

Comment:
@rraustad Are you intentionally adding the namespaces to these?  They aren't used that way in the other unit test in this file.  I'm in the midst of cleaning up so stray indentation changes in the main routine and will clean these up also if you agree.